src/implementor/token.py

```diff
@@ -4,10 +4,11 @@
 import json
 
 
-def create_token(body):
+def create_token(body, token):
```

Comment:
Same question here. Why does token creation need a token?

Reply:
Also as this is not implemented yet, how do you plan to generate a token. On this depends whether your decorator is safe or not.

Reply:
See above 😇

Reply:
What itches my brain is that you already have code to check a token but there is not code to generate one yet. Or did I overlook something?

I just wanted to get a feeling for the safety of our tokens. But I can also do that later when you plan to implement generation in another PR 🙂